airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -507,7 +507,7 @@ def create_http_requester(self, model: HttpRequesterModel, config: Config, **kwa
         )
 
         return HttpRequester(
-            name=model.name,
+            name=model.parameters.get("name", "") if model.parameters else "",
```

Comment:
does this mean the name has to be set in $parameters? This seems like a non-obvious restriction

Reply:
Changed the approach per the discussion below.